doc/source/how-to-use-secagg.rst

```diff
@@ -0,0 +1,150 @@
+Use Secure Aggregation
+======================
+
+What is Secure Aggregation?
+---------------------------
+
+Secure Aggregation is a class of Secure Multi-Party Computation algorithm that provides a way for an aggregated value (such as the sum of data) from multiple parties to be computed without revealing their individual inputs to one another.
```

Comment:
Should we use different terminology instead of "the sum of data"? For example, "the sum of parameters." This might help prevent users from thinking they are sending out data.

Reply:
Good point, I've updated it.